code/modules/shuttle/shuttle.dm

```diff
@@ -21,8 +21,6 @@
 	var/dwidth = 0	//position relative to covered area, perpendicular to dir
 	var/dheight = 0	//position relative to covered area, parallel to dir
 
-	shuttle_abstract_movable = 1
```

Comment:
Wait no yeah this is correct, stationary ports on the ship have to have the var changed in mapping.